en: update doc for alert and monitor (#83)
pingcap/docs-tidb-operator: en/monitor-a-tidb-cluster.md

When Prometheus is deployed with a TiDB cluster, some default alert rules are automatically imported. You can view all alert rules and statuses in the current system by accessing the Alerts page of Prometheus through a browser.

The custom configuration of alert rules is supported. You can modify the alert rules by taking the following steps:

1. When deploying the monitoring system for the TiDB cluster, set `spec.reloader.service.type` to `NodePort` or `LoadBalancer`. For details, see [Monitor the TiDB Cluster using TidbMonitor](monitor-using-tidbmonitor.md).
2. Access the `reloader` service through `NodePort` or `LoadBalancer`. Click the `Files` button above to select the alert rule file to be modified, and make the custom configuration. Click `Save` after the modification.

The default Prometheus and alert configuration do not support sending alert messages. To send an alert message, you can integrate Prometheus with any tool that supports Prometheus alerts. It is recommended to manage and send alert messages via [AlertManager](https://prometheus.io/docs/alerting/alertmanager/).

> - If you already have an available AlertManager service in your existing infrastructure, you can modify `spec.alertmanagerURL` and configure its address for use by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).

Reviewer: ```suggestion
- If you already have an available AlertManager service in your existing infrastructure, you can set the value of `spec.alertmanagerURL` to the address of `AlertManager`, which will be used by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
```

> - If you already have an available AlertManager service in your existing infrastructure, you can modify `spec.alertmanagerURL` and configure its address for use by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).

Author: ```suggestion
- If you already have an available AlertManager service in your existing infrastructure, you can set the value of `spec.alertmanagerURL` to the address of `AlertManager`, which will be used by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
```

> - If there is no AlertManager service available, or if you want to deploy a separate set of services, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).

Reviewer: ```suggestion
- If there is no AlertManager service available, or if you want to deploy a separate AlertManager service, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
```

> - If there is no AlertManager service available, or if you want to deploy a separate AlertManager service, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).

Reviewer 2: ```suggestion
- If no AlertManager service is available, or if you want to deploy a separate AlertManager service, refer to the [Prometheus official document](https://github.com/prometheus/alertmanager).
```
